package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lumber-cli",
-  "version": "2.5.1",
+  "version": "2.5.2",
```

Comment:
[misc] Wrong version, don't understand why it increases automatically...

Reply:
No the version is 2.5.2 🤔
 But now that you've said it the issue is that when we realease a new version we do not update the `package-lock` so the version is not updated

Reply:
Arnaud says we put yarn instead (As we all use it normally ^^)

Reply:
Yarn.lock does not have the version of the current package as it is useless ;)